### PR TITLE
[Docs] Full Node Documentation Update (#941 followup)

### PR DESCRIPTION
**Pull request by Olshansk**
## Summary

- Improvements to the Full Node walkthrough + cheatsheet
- A handful of other miscleanious changes to make the docs more "holistic"
- Prep for the validator cheatsheets
- 
## Issue

- Followup to #941

## Type of change

Select one or more from the following:

- [ ] New feature, functionality or library
- [ ] Consensus breaking; add the `consensus-breaking` label if so. See #791 for details
- [ ] Bug fix
- [ ] Code health or cleanup
- [x] Documentation
- [ ] Other (specify)

### CODE REVIEW

**Review by okdas — APPROVED** [review]
The changes look good. I haven't run the new script yet, but I'll be going to go through all of them again soon.

File: docusaurus/docs/operate/quickstart/full_node_cheatsheet.md [hunk 1]
```diff
@@ -43,24 +60,44 @@ To install and set up a Full Node, follow these steps:
...
+See the [FAQ & Troubleshooting section in the Full Node Walkthrough](../run_a_node/full_node_walkthrough.md#faq--troubleshooting)
+for examples of useful commands, common debugging instructions and other advanced usage.
+
+### [OPTIONAL] Do you care to know what just happened?
+
+:::info
+This section is optional and for informational purposes only.
+:::
+
+If you're interest in understand what just got installed, keep reading...
```
Comment (red-0ne) [inline docusaurus/docs/operate/quickstart/full_node_cheatsheet.md:89]:
```suggestion
If you're interested in understanding what just got installed, keep reading...
```

**Review by red-0ne — APPROVED** [review]
Great full node documentation :full_moon_with_face: 

Got a handful of change requests but nothing blocking merging.

Preemptively approving.

File: docusaurus/docs/operate/run_a_node/full_node_walkthrough.md [hunk 2]
```diff
@@ -201,52 +217,104 @@ sudo systemctl enable cosmovisor.service
 sudo systemctl start cosmovisor.service
```
[the remark this answers is not in the document]
Comment (Olshansk) [reply docusaurus/docs/operate/run_a_node/full_node_walkthrough.md:217]: Adding a TODO for @okdas

[the remark this answers is not in the document]
Comment (Olshansk) [reply docusaurus/docs/operate/run_a_node/full_node_walkthrough.md:217]: [`0a3d78d` (#981)](https://github.com/pokt-network/poktroll/pull/981/commits/0a3d78d4ded79bc15ca30888e27788b13a5ea5c0)

**Pull request merged**